matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/Session.kt

```diff
@@ -190,6 +191,11 @@ interface Session :
      */
     fun cryptoService(): CryptoService
 
+    /**
+     * Returns the ContentScanningService associated with the session
+     */
+    fun contentScanningService(): ContentScannerService
```

Comment:
`Scanning` / `Scanner`: this is maybe not new, but maybe an opportunity to use the same semantic everywhere.
I would vote for `Scanner`, as per https://github.com/matrix-org/matrix-content-scanner#matrix-content-scanner